### ISSUE TITLE
Where did FSharp.Compiler.CodeDom go?

### ISSUE THREAD

**Issue by JanBessai** [fsprojects-archive/zzarchive-powerpack#21]
Is FSharp.Compiler.CodeDom completely obsolete? In which successor project can I find it?
My usecase is xsd.exe together with CodeDom to create a F# type from an XSD. With FSharp.Data I only get XML to FSharp.

**Comment by robjens** [comment]
I patched https://xsd2code.codeplex.com community version to generate code for `4.0` and `4.5` plus (main reason ofc) `FSharp` codeprovider. Works like a charm but only in `CSharp` project though which, once generated will include it in the project and as such, have intellisense freak out because it cannot interpret any `.fs` files in a C# project. Anyhew, once I have some more time I'll have a look at it, seemed like the project was a tad outdated but these sort of growing pains in the F# tooling for VS might have been the reason though. Anyway, I'm happy since it generates the extension methods and all sorts of options I wouldn't have figured out in weeks probably. More than happy to share if anyone is interested. Oh, also had to update a Wix installer for 4.x but that wasn't too much of a pain, although I did have to disassemble the fsharp codeprovider, strong name sign with key, then reassemble and take that `.dll` (which I still had left from hacking fs codedom into the regular `xsd.exe`. For the rest easy pleasy =/

On a side note, anyone have an idea what https://github.com/janno-p/XmlSerializerTutorial/XmlSerializerTutorial/AnnaIsikuKvalifikatsioonid2.fs that guy used? I get a whole lot of different FSharp code every time. Manually hacking?
